### PR TITLE
Fullscreen toggle

### PR DESCRIPTION
**Pull request by not-surt**
Adds shortcut and button to toggle between fullscreen and windowed.

### CODE REVIEW

File: app/qml/ui/Shortcuts.qml [hunk 1]
```diff
@@ -11,7 +11,7 @@ Item {
     readonly property bool isImageProjectType: projectType === Project.ImageType || projectType === Project.LayeredImageType
     property Item canvasContainer
     property ImageCanvas canvas
-    readonly property bool canvasHasActiveFocus: canvas && canvas.activeFocus
+    readonly property bool canvasHasActiveFocus: canvas && canvas.activeFocus
```
Comment (mitchcurtis) [inline app/qml/ui/Shortcuts.qml]: Trailing space

File: app/qml/main.qml [hunk 2]
```diff
@@ -119,6 +120,18 @@ ApplicationWindow {
         }
     }
 
+    function toggleFullScreen() {
+        onActivated: {
```
Comment (mitchcurtis) [inline app/qml/main.qml]: What does onActivated mean here? The syntax looks odd; I'd remove it if it works without it.

**Comment by mitchcurtis** [comment]
Thanks for the patch! I was going to do a full review but it's easier to just push a follow-up patch. :)

File: app/qml/main.qml (hunk 2 above)
Comment (not-surt) [reply app/qml/main.qml]: Shouldn't be there, left over code when I had it directly inside the toolbutton.

Comment (not-surt) [reply app/qml/main.qml]: Fixed.

**Pull request merged**